misc.go

```diff
@@ -90,7 +91,7 @@ func statusChecker(status int) error {
 	}
 }
 
-func marshal(data interface{}, req *fiber.Request) (*fiber.Request, error) {
+func marshal(data any, req *fiber.Request) (*fiber.Request, error) {
```

Comment:
*fiber.Request in the return is unnecessary.

Reply:
Yes, correct. Nice catch!